erpnext/crm/doctype/contract/contract.js

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2018, Frappe Technologies Pvt. Ltd. and contributors
+// For license information, please see license.txt
+
+cur_frm.add_fetch("contract", "terms", "contract_terms");
```

Comment:
That's old API, there's a better way to do this now.

Reply:
Where should I look for the new way to do it?

Reply:
ERPNext code. I believe the new way is : `frm.blah`. @neilLasrado knows.

Reply:
That...didn't help it filter down. Google didn't help either.

Reply:
use frm.add_fetch

Reply:
That only works if it's in a function whose argument is `frm`.